### ISSUE TITLE
aria_hidden_text_input_custom vo_macos/safari

### ISSUE THREAD

**Issue by mfairchild365** [accessibilitysupported/a11ysupport.io#9]
This Support Point submission is for the test [aria-hidden on a text input](https://a11ysupport.io/tests/aria_hidden_text_input_custom)

| property | value |
| --- | --- |
| title | aria_hidden_text_input_custom |
| support | y |
| at | vo_macos |
| at_version | 10.13.6 |
| os_version | 10.13.6 |
| browser | safari |
| browser_version | 11.1.2 |
| output_1_command | VO+left/right arrows |
| output_1_command_name | Read previous/next item |
| output_1_output | (nothing) |
| output_1_result | pass |
| output_2_command | Tab |
| output_2_command_name | Next focusable element |
| output_2_output | (nothing) |
| output_2_result | pass |

== begin notes ==
When pressing tab, the hidden input does receive focus but is not announced in the traditional sense. Depending on the exact order of key-presses, VoiceOver will either announce the page title or that 'you are currently on a text field' (no label is announced). This is accurate because aria-hidden will not remove an element from the tab index.
== end notes ==
